### ISSUE TITLE
Internal service footer: update guidance to cover standard link inclusions

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#545]
## What
The internal service guidance needs an update to include what links should be included as standard (accessibility) and what some optional links could be depending on your service context (e.g.:privacy).

## Why
@petelonsdale (https://github.com/dwp/design-system/discussions/441#discussioncomment-7379695) raised a question that showed a gap in guidance. @htmlandbacon advised an accessibility statement should always be included.

## Done when

- [x] Update guidance to cover standard links (inclusions and context for optionals) (@martinwake)
- [x] Assess whether component (dwp-frontend) defaults include a 'accessibility' link name (@htmlandbacon)
- [x] Update guidance to include example links

**Comment by htmlandbacon** [comment]
Just as a bit of an outcome: suggestion is to give an example of the footer populated with the links, rather then hard code this as a 'thing' within the component.

This is inline with option components i.e [keydetails](https://design-system.dwp.gov.uk/components/key-details-bar).
